companion/src/modeledit/setup.cpp

```diff
@@ -695,6 +695,12 @@ void ModulePanel::update()
   ui->registrationIdLabel->setVisible(mask & MASK_ACCESS);
   ui->registrationId->setVisible(mask & MASK_ACCESS);
 
+  // fix 2.8 not writing yaml tag receiver:
+  for(uint8_t receiverIdx = 0; receiverIdx < 3; receiverIdx++) {
+    if(module.access.receiverName[receiverIdx][0])
+      module.access.receivers |= (1 << receiverIdx);
+  }
+
```

Comment:
This code gets called every time the tab receives focus. Far better to add fixup in the yaml module decode https://github.com/EdgeTX/edgetx/blob/main/companion/src/firmwares/edgetx/yaml_moduledata.cpp so only performed once and saves polluting the ui code. At some future version the fixup code should be removed.

Reply:
I would also update the comment to 'fix 2.8 color radios'

Reply:
done